packages/react/src/ActionList/Description.tsx

```diff
@@ -25,7 +24,7 @@ export type ActionListDescriptionProps = {
 
 export const Description: React.FC<React.PropsWithChildren<ActionListDescriptionProps>> = ({
   variant = 'inline',
-  sx = defaultSxProp,
+  sx,
```

Comment:
I removed the default assignment here because the BoxWithFallback component also will set `sx = defaultSxProp`